etc/profile-m-z/telegram.profile

```diff
@@ -36,10 +36,15 @@ noroot
 notv
 protocol unix,inet,inet6,netlink
 seccomp
+seccomp.block-secondary
 shell none
+tracelog
 
 disable-mnt
+#private-bin bash,sh,telegram-desktop
 private-cache
 private-dev
 private-etc alsa,alternatives,ca-certificates,crypto-policies,fonts,group,ld.so.cache,localtime,machine-id,os-release,passwd,pki,pulse,resolv.conf,ssl,xdg
 private-tmp
+
+dbus-system none
```

Comment:
I already thought about a `dbus-user filter` policy https://github.com/netblue30/firejail/issues/3638#issuecomment-695988562. Maybe you can try if the following works (still very weak).

```
dbus-user filter
dbus-user.own org.telegram.desktop
dbus-user.talk org.freedesktop.Notifications
dbus-user.talk org.kde.StatusNotifierWatcher
dbus-user.talk org.gnome.Mutter.IdleMonitor
dbus-user.talk org.freedesktop.ScreenSaver
```

Reply:
on kde I need just
```
dbus-user filter
dbus-user.talk org.freedesktop.Notifications
dbus-user.talk org.kde.StatusNotifierWatcher
```
btw, is `dbus-user.talk org.freedesktop.ScreenSaver` needed?

Reply:
IDK, they are just picked from it's flatpak, which has a lot of talk rules. It has even a talk rules for a `org.freedesktop.portal.foobar`, though flatpak always allows `org.freedesktop.portal.*`. If it is needed, it's used to inhibit the screensaver in e.g. video-calls.

Reply:
then we should allow bare minimum. & rest should be commented out. what are necessary on gnome for basic functioning.

Reply:
In general I agree with you to only allow a bare minimum. However there is always a difference between adding restrictions and weakening. Here we start introducing restrictions at all and I don't want break to much that was working before. Especial if you look at https://github.com/netblue30/firejail/wiki/Restrict-D-Bus. Allowing `org.freedesktop.ScreenSaver` if `org.freedesktop.Notifications` is already allowed adds virtually no new capabilities.